docs/python_support/python_support.md

```diff
@@ -0,0 +1,333 @@
+# Python support in OpenVINO Model Server - preview {#ovms_docs_python_support_python_support}
+
+
+5. PythonExecutorCalculator And Mediapipe Graphs
+6. Usage With gRPC Streaming
+> timestamping and output gathering
+> single request multiple response (cycles)
+
+## Introduction
+
+**This feature is in preview, meaning some behviors of the feature as well as user interface are subjects to change in the future versions**
+
+ Starting with version 2023.3, model server introduced support for execution of custom Python code. Users can now create their own servables in Python. Those servables can run simple pre or post processing tasks as well as complex ones like image or text generation. 
+ 
+ Python is enabled via [Mediapipe](../mediapipe.md) by built-in `PythonExecutorCalculator` that allows creating graph nodes that execute Python code. That way Python servables can be used as standalone endpoints (single node graphs) or be part of larger Mediapipe solutions.
```

Comment:
Minor: mediapipe is refered everywhere with 2 capital letters - MediaPipe, maybe we should do it as well?
https://developers.google.com/mediapipe